pkg/scheduler/framework/runtime/framework.go

```diff
@@ -126,12 +126,13 @@ func (f *frameworkImpl) getExtensionPoints(plugins *config.Plugins) []extensionP
 		{plugins.PostFilter, &f.postFilterPlugins},
 		{plugins.Reserve, &f.reservePlugins},
 		{plugins.PreScore, &f.preScorePlugins},
-		{plugins.Score, &f.scorePlugins},
 		{plugins.PreBind, &f.preBindPlugins},
 		{plugins.Bind, &f.bindPlugins},
 		{plugins.PostBind, &f.postBindPlugins},
 		{plugins.Permit, &f.permitPlugins},
 		{plugins.QueueSort, &f.queueSortPlugins},
+		// Keep plugins.Score at the end for weighting.
```

Comment:
can you pls say something here for why keep the plugin at the end?  maybe link to the issue or just explain a little bit here?